ui/packages/consul-ui/app/templates/dc/services/instance/healthchecks.hbs

```diff
@@ -27,7 +27,7 @@ as |route|>
       )
     )
 
-    route.model.item.MeshChecks
+    (merge-checks (array route.model.item.Checks route.model.proxy.Checks) route.model.proxy.ServiceProxy.Expose.Checks)
```

Comment:
So where we merge all the checks (those from the ServiceInstance and the Proxy) together before iterating through them. This uses the exact same code as previously, only we do it in the template instead of in the model which then avoids all manner of problems.